ts/src/probit.ts

```diff
@@ -1200,21 +1203,22 @@ export default class probit extends Exchange {
     }
 
     costToPrecision (symbol, cost) {
-        return this.decimalToPrecision (cost, TRUNCATE, this.markets[symbol]['precision']['cost'], this.precisionMode);
+        return this.decimalToPrecision (cost, TRUNCATE, this.markets[symbol]['info']['cost_precision'], this.precisionMode);
```

Comment:
@Dan-krm I don't think this edit is correct, because it will return a precision not compatible with the precision mode. Going to revert it but let me know if antyhing